MapboxNavigation/Simulation.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import CoreLocation
+
+protocol SimulationDelegate : class {
+    func simulation(_ locationManager: CLLocationManager, didUpdateLocations locations: [CLLocation])
+}
+
+class Simulation : NSObject {
```

Comment:
I think “SimulatedRoute” would make the nature of this class clearer. “Simulation” could be a system for simulating routes.